src/redux/news/news.operations.js

```diff
@@ -0,0 +1,173 @@
+import { gql } from 'apollo-boost';
+import { client } from '../../utils/client';
+
+const getAllNews = () =>
+  client
```

Comment:
Let's use one unified way for the operations - `async - await`. Refactor code from `then` usages to the `async - await` usages

Reply:
Hope I understood correctly. Changed!